core/exceptions/ExceptionStackTraceDisplay.php

```diff
@@ -416,18 +361,18 @@ private function printScripts($force_print = false)
                 return '';
             }
         }
-        $jquery = esc_url_raw(includes_url() . 'js/jquery/jquery.js');
-        $core = esc_url_raw(EE_GLOBAL_ASSETS_URL . 'scripts/espresso_core.js?ver=' . espresso_version());
-        $ee_error = esc_url_raw(EE_GLOBAL_ASSETS_URL . 'scripts/EE_Error.js?ver=' . espresso_version());
+        $jquery = includes_url() . 'js/jquery/jquery.js';
+        $core = EE_GLOBAL_ASSETS_URL . 'scripts/espresso_core.js?ver=' . espresso_version();
+        $ee_error = EE_GLOBAL_ASSETS_URL . 'scripts/EE_Error.js?ver=' . espresso_version();
+        $style = EE_GLOBAL_ASSETS_URL . 'css/ee-exception-stack-display.css';
         return '
-<script>
-/* <![CDATA[ */
```

Comment:
Removed the CDATA because `kses` considered it as an unknown HTML tag and removed the tag and its contents.